truss/remote/baseten/core.py

```diff
@@ -47,18 +47,43 @@ def exists_model(api: BasetenApi, model_name: str) -> Optional[str]:
     return None
 
 
-def get_model_versions_info(api: BasetenApi, model_name) -> Tuple[str, dict]:
-    query_result = api.get_model(model_name)["model_version"]["oracle"]
+def get_model_versions_info(api: BasetenApi, model_name: ModelName) -> Tuple[str, List]:
+    query_result = api.get_model(model_name.value)["model"]
     return (query_result["id"], query_result["versions"])
 
 
+def get_model_versions_info_by_id(
```

Comment:
A lot of these functions have `info` which I think doesn't add a lot, I think this could just be `def get_model_versions_by_id`